### DISCUSSION TITLE
How to use the MiXCR software to analyze incomplete rearrangements such as IGH-DJ and IGKV-Kde?

### DISCUSSION THREAD

**Discussion by tongyuBGI** [milaboratory/mixcr#1607] [Q&A] [answered]
How can I use the MiXCR software to analyze incomplete rearrangements like IGH-DJ and IGKV-Kde? I would like to know if I need to create my own reference database. How should I establish this database?

thanks!

**Comment by mizraelson** [comment, marked as answer]
Hi, the latest MiXCR version supports incomplete rearrangements.  Please write to support@milaboratories.com for more detail.
